snippets/xml_snippets.json

```diff
@@ -276,6 +276,12 @@
         ],
         "description": "Add field tag"
     },
+    "Filter tag": {
```

Comment:
Seems you forgot `description` key for this new snippet. Also, you have to add it in the Documentation table So, users can know snippets like this exist.